src/app/helptext/system/ssh-connections.ts

```diff
@@ -0,0 +1,38 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    setup_method_placeholder: T('Setup Method'),
+    setup_method_tooltip: T(''),
+
+    host_placeholder: T('Host'),
+    host_tooltip: T(''),
+
+    port_placeholder: T('Port'),
+    port_tooltip: T(''),
+
+    username_placeholder: T('Username'),
+    username_tooltip: T(''),
+
+    private_key_placeholder: T('Private Key'),
+    private_key_tooltip: T(''),
+
+    remote_host_key_placeholder: T('Remote Host Key'),
+    remote_host_key_tooltip: T(''),
+
+    cipher_placeholder: T('Cipher'),
+    cipher_tooltip: T(''),
```

Comment:
From Replication Task Options table:

Encryption algorithm to secure the connection. <i>Standard</i> provides the best security. <i>Fast</i> is less secure, but has better transfer rates for devices with limited cryptographic speed. <i>Disabled</i> is for networks where the entire path between sources and destinations is trusted.